### DISCUSSION TITLE
GeoIP ?

### DISCUSSION THREAD

**Discussion by gingerlime** [nginx-proxy/nginx-proxy#2240] [Q&A]
Thanks for nginx-proxy. It's really great!

I'm wondering if anyone can share any "recipes" for adding GeoIP support for nginx using nginx-proxy? I'm trying to work out the best setup to let nginx work out the geoip country and add it to the uppstream via a header...

**Comment by dweipert-3138720606** [comment]
You would have to build the nginx-proxy image yourself.

But it seems you also have to build yourself an nginx image since GeoIP is not enabled by default and I can't see any configuration options or other images supporting non-standard libraries here: https://hub.docker.com/_/nginx/
